### PR TITLE
Reorg build files

### PR DESCRIPTION
**Pull request by pieces029**
Updated the build files and moved common dependencies to the top level build file.

### CODE REVIEW

File: SwissKnife/gradlew.bat [hunk 1]
```diff
@@ -1,90 +0,0 @@
-@if "%DEBUG%" == "" @echo off
```
Comment (mg6maciej) [inline SwissKnife/gradlew.bat]: Why you no like Windows users? ;)

Comment (pieces029) [inline SwissKnife/gradlew.bat]: This was in the wrong folder. There is one in the top level.

Comment (mg6maciej) [inline SwissKnife/gradlew.bat]: Oh, ok. I meant that as a joke.

Comment (pieces029) [inline SwissKnife/gradlew.bat]: :)

**Pull request merged**